src/util/RouterUtil.js

```diff
@@ -133,17 +133,12 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
 
       var redirectFn = router.settings.get('redirectUtilFn');
 
-      if (res.type === Enums.SESSION_STEP_UP) {
-        var targetUrl = res._links && res._links.next && res._links.next.href;
-        successData.stepUp = {
-          url: targetUrl,
-          finish: function () {
-            redirectFn(targetUrl);
-          }
+      var nextUrl = res._links && res._links.next && res._links.next.href;
+      if (nextUrl) {
+        successData.next = function () {
+          redirectFn(nextUrl);
         };
       } else {
-        // Add the type for now until the API returns it.
-        successData.type = Enums.SESSION_SSO;
```

Comment:
just making sure it's entirely safe to remove this, though I also assumed it wasn't used for anything.

Reply:
@royalchan-okta I haven't removed, it's here: https://github.com/okta/okta-signin-widget/pull/619/files#diff-1d1f2f1a166e47fd617d3b5e9dfe4b74R127 ^^
However, that's a good point! Should we remove it? Is the API now guaranteed to always return a res.type?

Reply:
no, unfortunately not, as some older flows will omit it because it isn't even set in the state token in okta-core